### DISCUSSION TITLE
UseApiVersioning() gone in Microsoft.AspNetCore.Mvc.Versioning 1.0.1+

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#187] [Q&A]
﻿**Which project template is the issue about?**
API
...

**Which version of the project template is the issue about?**
Latest on master (not sure where to find the version number for the template)
...

**Which version of Visual Studio or the dotnet SDK were you using?**
2.1.4
...

I see were using an RC version of Microsoft.AspNetCore.Mvc.Versioning.ApiExplorer and that this call is gone.

Also is there any plans to update to target netcoreapp2.0 instead of 1.0?

**Comment by RehanSaeed** [comment]
Updated and released 1.7.0 to NuGet.
